src/style.css

```diff
@@ -1,3 +1,83 @@
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+  width: 100%;
+}
+
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  margin: 0;
+}
+
+.logo-img {
+  height: 40px;
+  width: 40px;
+  display: flex;
+  justify-content: center;
+}
+
+.ul-decor {
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  margin: 0;
+  padding: 0;
+  align-items: center;
+}
+
+.li-set{
+  margin-left: 20px;
+  display: flex;
+  align-items: center;
+}
+
+li:first-child {
+  margin-left: 0;
+}
+
+a {
```

Comment:
Don't use tag names for styling (except html and body) ([checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md))